kin/blockchain/channel_manager.py

```diff
@@ -57,11 +58,19 @@ def build_transaction(self, add_ops_fn, memo_text=None):
         source = self.base_address if builder.address != self.base_address else None
 
         # add operation (using external partial) and sign
-        add_ops_fn(builder)(source=source)
-        if memo_text:
-            builder.add_text_memo(memo_text)  # max memo length is 28
-
-        builder.sign()  # always sign with a channel key
-        if source:
-            builder.sign(secret=self.base_key)  # sign with the base key if needed
+        try:
+            add_ops_fn(builder)(source=source)
+            # update fee
```

Comment:
redundant comment (if there's a special reason why that needed to be mentioned explain it o.w. it speaks for itself)